packages/shared/trade/src/trade-action-summary/trade-action-summary.tsx

```diff
@@ -127,6 +127,9 @@ export function TradeActionSummary({
           }
         />
         <TradeActionTitle value={totalAPY} title={apySuffix} valueSuffix="%" />
+        {liquidityYieldData && (
+          <LiquidityYieldInfo liquidityYieldData={liquidityYieldData} />
+        )}
         {isLeveraged && customizeLeverage && (
```

Comment:
Does the formula show up on the Manage Leveraged Liquidity actions, like Adjust Leverage, Withdraw, and Roll Maturity?